modules/installation-gcp-permissions.adoc

```diff
@@ -336,53 +40,29 @@ ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
 * Deployment Manager Editor
 * Service Account Key Admin
```

Comment:
Per the permissions section `Required permissions for creating Service Account resources` [1], I think we should move the role `Service Account Key Admin` to the section `Required roles for the installation program` [2] instead, for both IPI and UPI. Thanks!  

[1] the related permissions listed in `Required permissions for creating Service Account resources`:
- iam.serviceAccountKeys.create
- iam.serviceAccountKeys.delete
- iam.serviceAccountKeys.get
- iam.serviceAccountKeys.list

[2] none existing role listed in `Required roles for the installation program` have all the above permissions

[jiwei@jiwei ~]$ gcloud iam roles describe **roles/compute.admin** | grep iam.serviceAccountKeys
[jiwei@jiwei ~]$ gcloud iam roles describe **roles/iam.securityAdmin** | grep iam.serviceAccountKeys
\- iam.serviceAccountKeys.list
[jiwei@jiwei ~]$ gcloud iam roles describe **roles/iam.serviceAccountAdmin** | grep iam.serviceAccountKeys
[jiwei@jiwei ~]$ gcloud iam roles describe **roles/iam.serviceAccountUser** | grep iam.serviceAccountKeys
[jiwei@jiwei ~]$ gcloud iam roles describe **roles/storage.admin** | grep iam.serviceAccountKeys
[jiwei@jiwei ~]$ 

_FYI_
[jiwei@jiwei ~]$ gcloud iam roles describe **roles/iam.serviceAccountKeyAdmin** | grep iam.serviceAccountKeys
\- iam.serviceAccountKeys.create
\- iam.serviceAccountKeys.delete
\- iam.serviceAccountKeys.disable
\- iam.serviceAccountKeys.enable
\- iam.serviceAccountKeys.get
\- iam.serviceAccountKeys.list
[jiwei@jiwei ~]$